lib/src/js_proxies/js_object_map_proxy.dart

```diff
@@ -14,7 +14,7 @@ class JsObjectMapProxy<V> with MapMixin<String, V> {
   Iterable<String> get keys => interop.Object.keys(_jsObject).cast<String>();
 
   @override
-  V operator [](Object key) => jsDecode(getProperty(_jsObject, key));
+  V? operator [](Object? key) => jsDecode(getProperty(_jsObject, key!));
```

Comment:
If Dart lets you, `key` should not be nullable. If it must be nullable, this should throw an `ArgumentError` (using `ArgumentError.notNull` or similar preferably) when `key` is null instead of using `!`.

Reply:
sadly it does not. `MapMixin` declares a nullable key now

Reply:
Ah, that's unfortunate. Yeah, this definitely should manually throw an `ArgumentError` when `key` is null then if `getProperty` can't take a nullable key itself.